modules/workflow-condition-parser/src/main/antlr4/org/opencastproject/workflow/conditionparser/antlr/WorkflowCondition.g4

```diff
@@ -10,7 +10,7 @@ atom : '(' relationOperand ')' | NUMBER | STRING | BOOL ;
 AND: 'AND' ;
 OR: 'OR' ;
 NOT: 'NOT' ;
-NUMBER: '-'? [0-9]+ ('.' [0-9]+)? ;
+NUMBER: '-'? [0-9]+ ('.' [0-9]+)? ( [E|e] [+\-]? [0-9]+ )? ;
```

Comment:
If you actually want to parse what Java allows you to parse, you might also want to allow a `+` sign in front of the first number:
```suggestion
NUMBER: [+\-]? [0-9]+ ('.' [0-9]+)? ( [E|e] [+\-]? [0-9]+ )? ;
```